pulp_container/app/tasks/sync_stages.py

```diff
@@ -140,6 +162,9 @@ async def run(self):
                     await self.put(man_dc)
                     tag_dc.extra_data["tagged_manifest_dc"] = man_dc
                     await self.handle_blobs(man_dc, content_data)
+                    if signature_source is not None:
+                        signature_dcs.extend(await self.create_signatures(man_dc, signature_source))
```

Comment:
I am not sure whether our rh registry contains any manifest lists to test, but we should pull in signatures for manifest lists as well.
The question that we need to answer is whether to pull in signatures per every image manifest that's in that manifest list or not. Maybe in reality only manifest list is signed, I asked Miroslav on more details.

Reply:
Here is what I learned- the image verification code only cares about signatures on the per-arch images; manifest lists might have signatures but they don’t really matter.
So, we should for sure  mirror signatures for every manifest that is mirrored, incl. the subordinate per-arch ones.

Reply:
Thanks for asking Miroslav. I took the docs as it's only supported for images https://github.com/containers/image/blob/main/docs/containers-signature.5.md#high-level-overview. So I made clear almost everywhere that it for images only, incl. docstrings.

I understand that at tech level, it should be the same. It affects tests and general expectations.